demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3580 @@
+{
```

Comment:
<blockquote>so we'll finish this demo covering some example steps to go data in files to a&nbsp;<code>StellarGraph</code>.</blockquote><p>I think there might be a wrong/missing word from what you intended - "go through data" maybe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>"go from" I think was what I meant.</p>